addon/components/paper-switch.js

```diff
@@ -84,12 +92,13 @@ export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
   },
```

Comment:
Safestring yes, but you don't need triple curlies.

Example:
  - https://github.com/miguelcobain/ember-paper/blob/wip/v1.0.0/addon%2Fcomponents%2Fpaper-progress-circular.js#L43-L49
  - https://github.com/miguelcobain/ember-paper/blob/wip/v1.0.0/app%2Ftemplates%2Fcomponents%2Fpaper-progress-circular.hbs#L1